src/modules/AddOnProduct.rb

```diff
@@ -818,8 +818,10 @@ def RegisterAddOnProduct(src_id)
     # Do installation of the add-on product within an installed system
     # srcid is got via AddOnProduct::src_id
     #
+    # @param install_packages [Boolean] install the selected packages,
+    #   can be optionally disabled
     # @return [Symbol] the result symbol from wizard sequencer
-    def DoInstall
+    def DoInstall(install_packages = true)
```

Comment:
what about using named argument? In general I do not like much boolean params. Also documentation is a bit lacking as this happen only when it missing installation.xml, not?